promkit-derive/examples/derive.rs

```diff
@@ -0,0 +1,24 @@
+use promkit::{crossterm::style::Color, style::StyleBuilder};
+use promkit_derive::Promkit;
+
+#[derive(Default, Debug, Promkit)]
+struct Profile {
+    #[form(
+        prefix = "What is your name?",
+        prefix_style = StyleBuilder::new().fgc(Color::DarkCyan).build(),
```

Comment:
I think `prefix` could still be confusing here, as it usually means what to apply before a given thing. I would propose:

```rust
#[form(
    prompt = "What is your name?",
    prompt_style = StyleBuilder::new().fgc(Color::DarkCyan).build(),
)]
```

or

```rust
#[form(
    label = "What is your name?",
    label_style = StyleBuilder::new().fgc(Color::DarkCyan).build(),
)]
```

or even

```rust
#[form(
    question = "What is your name?",
    question_style = StyleBuilder::new().fgc(Color::DarkCyan).build(),
)]
```

Reply:
Adopted `label` at [c6a9593](https://github.com/ynqa/promkit/pull/17/commits/c6a95930a6ce46e8ef6781d784cd613df9e2dde7).

Reply:
@simonsan Adopted `label` at [c6a9593](https://github.com/ynqa/promkit/pull/17/commits/c6a95930a6ce46e8ef6781d784cd613df9e2dde7).